src/page-multi-facility/LocaleDetails.tsx

```diff
@@ -0,0 +1,86 @@
+import numeral from "numeral";
+import React from "react";
+import styled from "styled-components";
+
+import Colors from "../design-system/Colors";
+import {
+  LocaleDataProvider,
+  LocaleRecord,
+  useLocaleDataState,
+} from "../locale-data-context/LocaleDataContext";
+
+const LocaleDetailsContainer = styled.div`
+  color: ${Colors.green};
+  display: flex;
+  flex-flow: row nowrap;
+  line-height: 150%;
+  letter-spacing: -0.02em;
+  margin: 2em 0;
+`;
+
+const LocaleDetail = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin-right: 32px;
+`;
+
+const LocaleDetailValue = styled.div`
+  font-family: "Poppins", sans serif;
+  font-size: 14px;
+  font-weight: 600;
+  margin-bottom: 8px;
+`;
+
+const LocaleDetailDescription = styled.div`
+  font-family: "Helvetica";
+  font-size: 13px;
+  font-weight: normal;
+`;
+
+const localeData: { [key: string]: string | undefined } = {
+  "Population": "totalIncarceratedPopulation",
+  "Cases": "reportedCases",
+  "Likely Cases": "estimatedIncarceratedCases",
+  "Hospital Beds": "hospitalBeds",
+  "ICU Beds": undefined,
+};
```

Comment:
I think another way you could do this that would be a bit more type-safe (so you wouldn't have to make the type assertion on L68 below) would be to make this typescript string enum instead. then use that enum to define the keys on `LocaleRecord` so that they are guaranteed to be the same. (i skipped some details in there because I am a little bit fuzzy on them honestly ... just get in touch if you want to discuss in more detail )

Reply:
@macfarlandian I spent a couple hours learning about string enums and trying to set something up that makes sense, but hitting some walls. Let me know if you have some time tomorrow to look at it together! 

Here's a snippet of where I've gotten to so far...

```
enum LocaleRecordProperties {
  county = "county",
  estimatedIncarceratedCases = "estimatedIncarceratedCases",
  hospitalBeds = "hospitalBeds",
  reportedCases = "reportedCases",
  state = "state",
  totalIncarceratedPopulation = "totalIncarceratedPopulation",
  icuBeds = "icuBeds"
}

type LocaleRecord = {
  [LocaleRecordProperties.county]: string;
  [LocaleRecordProperties.estimatedIncarceratedCases]: number;
  [LocaleRecordProperties.hospitalBeds]: number;
  [LocaleRecordProperties.reportedCases]: number;
  [LocaleRecordProperties.state]: string;
  [LocaleRecordProperties.totalIncarceratedPopulation]: number;
  [LocaleRecordProperties.icuBeds]: number;
};

const localePropertyHeaders: { [key: string]: string | undefined } = {
  [LocaleRecordProperties.totalIncarceratedPopulation]: "Population",
  [LocaleRecordProperties.reportedCases]: "Cases",
  [LocaleRecordProperties.estimatedIncarceratedCases]: "Likely Cases",
  [LocaleRecordProperties.hospitalBeds]: "Hospital Beds",
  [LocaleRecordProperties.icuBeds]: "ICU Beds",
};

// Here I get an error at localeRecord[property] that 
// I can't use type string to index type LocaleRecord. 
// I tried to set a type for the property argument in the 
// map callback to `keyof typeof LocaleRecordProperties`, 
// but that returns a union type of the keys and then I see an 
// error on the argument that I can't assign a union type to a string argument. 
Object.keys(localePropertyHeaders).map(property => { 
    const description = localePropertyHeaders[property];
    const value = localeRecord[property]
              ? numeral(localeRecord[property]).format("0,0")
              : enDash;
}

```